docs/products/kafka/howto/enable-kafka-tiered-storage.md

```diff
@@ -94,42 +73,78 @@ Kafka service using the [Aiven CLI](/docs/tools/cli):
-    use this command:
+1. To find the name of the Aiven for Apache Kafka service for enabling tiered storage,
+   use this command:
 
-    ```bash
-    avn service list
-    ```
+   ```bash
+   avn service list
+   ```
 
-    Make a note of the `SERVICE_NAME` corresponding to your Aiven for
-    Apache Kafka service.
+   Make a note of the `SERVICE_NAME` corresponding to your Aiven for
+   Apache Kafka service.
 
-1.  Enable tiered storage using the command below:
+1. Enable tiered storage using the command below:
 
-    ```bash
-    avn service update \
-       --project demo-kafka-project \
-       demo-kafka-service \
-       -c tiered_storage.enabled=true
-    ```
+   ```bash
+   avn service update \
+      --project demo-kafka-project \
+      demo-kafka-service \
+      -c tiered_storage.enabled=true
+   ```
 
 In this command:
 
--   `--project demo-kafka-project`: Replace `demo-kafka-project` with
-    your project name.
--   `demo-kafka-service`: Specify the Aiven for Apache Kafka service you
-    intend to update.
--   `-c tiered_storage.enabled=true`: Configuration flag that activates
-    tiered storage for your Aiven for Apache Kafka service.
+-  `--project demo-kafka-project`: Replace `demo-kafka-project` with
+   your project name.
+-  `demo-kafka-service`: Specify the Aiven for Apache Kafka service you
+   intend to update.
+-  `-c tiered_storage.enabled=true`: Configuration flag that activates
+   tiered storage for your Aiven for Apache Kafka service.
+
+</TabItem>
+</Tabs>
+
+## Configure default retention policies at service-level
+
+To efficiently manage data retention, set default policies for tiered storage at
+the service level.
+
+1. Access [Aiven console](https://console.aiven.io/), select your
+   project, and choose your Aiven for Apache Kafka service.
+1. Click <ConsoleLabel name="service settings"/> from the sidebar.
+1. Scroll down to the **Advanced configuration** section, and click **Configure**.
+1. In the **Advanced configuration** dialog, click <ConsoleLabel name="addadvancedconfiguration" />.
+1. To define the retention policy for Aiven for Apache Kafka tiered
+   storage, choose either of these options:
+   -   Find `kafka.log_local_retention_ms` and set the value to define
+       the retention period in milliseconds for time-based retention.
+   -   Find `kafka.log_local_retention_bytes` and set the value to
+       define the retention limit in bytes for size-based retention.
+1. Click **Save configuration** to apply your changes.
+
+Additionally, you can configure the retention policies from the
+[Tiered storage overview](/docs/products/kafka/howto/tiered-storage-overview-page#modify-retention-polices) page.
+
+## (Optional) configure client-side parameter
+
+For optimal performance and reduced risk of broker interruptions when
+using tiered storage, it is recommended to update the client-side
+parameter `fetch.max.wait.ms` from its default value of 500 ms to 5000 ms.
```

Comment:
I guess make sense to have this repeated. Apply same suggestions as previous one